### DISCUSSION TITLE
Entrega de practicas

### DISCUSSION THREAD

**Discussion by 0s4k4** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#44] [Q&A]
Hola comunidad, saben como se van entregar las practicas que dejo el sherpaGOD? gracias por su atencion uwu

**Comment by PaulHRios** [comment]
Parece ser que las subiremos a un repositorio. En la sesión de la noche podremos confirmarlo.

**Comment by GloriaCoralCerecedo** [comment]
¿Creen que den más chance para entregar las prácticas?

**Reply by RafBD** [reply]
Espero que sí, porque muchos no entendemos que hay que hacer. :(
